Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM python:3.7-alpine
 
 ENV ZOO_RDS_PRODUCTION_CERT_PATH=/etc/ssl/certs/rds-combined-ca-bundle.pem \
+    ZOO_DATABASE_OPTIONS=sslmode=verify-full,sslrootpath=/etc/ssl/certs/rds-combined-ca-bundle.pem \
```

Comment:
And this as well, this should go in the final image, so this can be reused with different configs without many changes.